### PR TITLE
Adds the ENS registry for goerli testnet

### PR DESCRIPTION
**Pull request by manuelwedler**
## Description

The ENS registry for goerli testnet has been added to version 1.x but is still missing in 2.x. This PR adds it in branch 2.x.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran ```npm run dtslint``` with success and extended the tests and types if necessary.
- [ ] I ran ```npm run test:unit``` with success and extended the tests if necessary.
- [ ] I ran ```npm run build-all``` and tested the resulting file/'s from  ```dist``` folder in a browser.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the live network.

### CODE REVIEW

**Comment by nivida** [comment]
@manuelwedler The development of 2.x is currently stale because of the limited resources we have. I apologize already for the delay until this PR is reviewed, merged and released. The final 2.x architecture I've worked out can provide the "legacy" API and the new functional API backed from the same source. I'm currently not sure how we will handle this but the goal would be to slowly deprecate the "legacy" API in 2.x as soon as it is released as a [non-prerelease version](https://semver.org/) (Just in case you are scared about the 2.x refactoring PR here on GitHub).

**Comment by manuelwedler** [comment]
Alright. Thanks for letting me know!

**Review by ryanio — APPROVED** [review]
LGTM, thanks! I am not sure when we may prepare a next 2.x release but good to get this in.

**Pull request merged**
